nima/http2/webserver/src/main/java/module-info.java

```diff
@@ -14,12 +14,21 @@
  * limitations under the License.
  */
 
+import io.helidon.common.features.api.Feature;
+import io.helidon.common.features.api.HelidonFlavor;
 import io.helidon.nima.webserver.spi.ServerConnectionProvider;
 
 /**
  * HTTP/2 WebServer.
  */
+@Feature(value = "Http2Webserver",
+        description = "HTTP/2 WebServer",
+        in = HelidonFlavor.NIMA, invalidIn = HelidonFlavor.SE,
+        path = {"WebServer","HTTP/2"}
```

Comment:
path is `HTTP/2", "WebServer"

this is a bit different than in reactive, as here http2 is the higher feature